src/addons/jest-matchers.js

```diff
@@ -18,6 +18,11 @@ expect.extend({
   },
 
   toHaveStatus (action, status) {
+    if (action instanceof Action === false) {
+      throw new TypeError("toHaveStatus expects an Action. Received " +
+                          (action ? "a " + action.constructor.name : action) + ".")
```

Comment:
Ah.. hmm. I see your point though. It'll be confusing to read:

```
expect("foobar").toHaveStatus("closed")
toHaveStatus expects an Action. Received "foobar".
```

Instead of:

```
"toHaveStatus expects an Action. Received a String."
```

What's better in your opinion?